packages/box/src/Box.tsx

```diff
@@ -49,11 +50,20 @@ const shouldForwardProp = createShouldForwardProp([
  *
  * @param props any
  */
-const sx = (props: any): SerializedStyles => {
+function sx(props: any): SerializedStyles {
   return css(props.sx)(props.theme)
 }
 
-type BoxProps = { as?: React.ElementType } & SpaceProps &
+function variant(props: any) {
```

Comment:
can we use this?
```
type VariantProps = {
  theme: Theme
  variant?: string
}
```
and `{ Theme } from 'theme-ui'`

Reply:
I tried that and something was bombing with it but I'll try it again and see if it works.

Reply:
Can we also export this since we'll probably have to re-use it?

Reply:
Yep!

Reply:
Types are fixed and exported both the interface and the `variant` function.